File: tests/dynamic_fixtures/go/xss_adversarial.go
// XSS — adversarial collision fixture.
// Prints the XSS oracle marker unconditionally without rendering any template
// and without emitting __NYX_SINK_HIT__.
// Expected verdict: Inconclusive(OracleCollisionSuspected)
// Entry: RenderPage(userInput string)  Cap: HTML_ESCAPE

package entry

import "fmt"

func RenderPage(userInput string) {
	// Coincidental oracle match — not an HTML render sink.
	fmt.Println("<script>NYX_XSS_CONFIRMED</script>")
	_ = len(userInput)
}

File: tests/dynamic_fixtures/python/xss_adversarial.py
"""XSS — adversarial collision fixture.

Outputs the XSS marker string unconditionally without it being a real
HTML sink (e.g., a test that checks for a string literal).
Expected verdict: Inconclusive(OracleCollisionSuspected).
"""


def render_comment(user_input):
    """Prints oracle marker outside of any HTML rendering context."""
    # Coincidental match — not an HTML sink.
    print("<script>NYX_XSS_CONFIRMED</script>")
    return user_input
